test/IntegrationTests/TicketService/test_create_ticket_by_conversation.py

```diff
@@ -0,0 +1,346 @@
+import os
+import sys
+import unittest
+import requests
+import json # Added for json.dumps
+from http import HTTPStatus
+
+# Ensure the utils directory is in the Python path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../"))) # Adjust path to root of test utils
+from utils.api_utils import add_customer, create_test_token, create_token_without_org_id, create_token_without_org_role
+
+from src.backend.lib.logging_config import get_primitivechat_logger # Path to actual logger
+logger = get_primitivechat_logger(__name__)
+
+
+class TestCreateTicketByConversationAPI(unittest.TestCase):
+    BASE_URL = f"http://{os.getenv('CHAT_SERVICE_HOST', 'localhost')}:{os.getenv('CHAT_SERVICE_PORT', '8000')}" # Provide defaults
+    ORG_ROLE = 'org:admin' 
+    allowed_custom_field_sql_types = ["VARCHAR(255)", "INT", "BOOLEAN", "DATETIME", "MEDIUMTEXT", "FLOAT", "TEXT"]
```

Comment:
Since we are removing custom field integrations. You can remove this line and also custom field related testcases